src/longsequences/transformations.jl

```diff
@@ -59,22 +59,62 @@ end
 
 Reverse a biological sequence `seq` in place.
 """
-function Base.reverse!(seq::LongSequence)
-    orphan!(seq) # TODO: Is the orphan call really nessecery given the indexing calls will also call orphan?
-    @inbounds for i in 1:div(lastindex(seq), 2)
-	    x = seq[i]
-        i′ = lastindex(seq) - i + 1 
-	    seq[i] = seq[i′]
-	    seq[i′] = x
+Base.reverse!(seq::LongSequence{<:Alphabet}) = _reverse!(orphan!(seq), BitsPerSymbol(seq))
+
+function _reverse!(seq::LongSequence{A}, ::BitsPerSymbol) where {A <: Alphabet}
+    i, j = 1, lastindex(seq)
+    while i < j
```

Comment:
Like the generic fallback kernel. Can we make this a bit faster by using `@inbounds`?

Reply:
Yes, that should happen.